_includes/articles/payment-batching.md

```diff
@@ -156,9 +153,10 @@ transaction group containing your transaction.
 
 The closer to a limit a transaction group becomes, the less likely
```

Comment:
I think it'd be good to have a brief explanation of what the limits are (size and number of connected transactions) before this paragraph.